Dockerfile

```diff
@@ -44,7 +44,8 @@ RUN \
   tar xf \
   /tmp/ddclient.tar.gz -C \
     /tmp/ddclient --strip-components=1 && \
-  install -Dm755 /tmp/ddclient/ddclient.in /usr/bin/ && \
+  cp /tmp/ddclient/ddclient.in /usr/bin/ddclient.in && \
+  ln -s /usr/bin/ddclient.in /usr/bin/ddclient && \
```

Comment:
Should we be chmodding the file in case they screw up upstream and don't?

Reply:
It probably won't be an issue for another 2 years :p I'll look into it in more depth when I'm on the desktop. I'll merge this now because the hourly failing external trigger build is driving me nuts.